common/app/model/dotcomrendering/pageElements/PageElement.scala

```diff
@@ -71,7 +74,9 @@ object Sponsorship {
   }
 }
 
+//noinspection ScalaStyle
```

Comment:
I just boosted the max complexity here so we might want to change it back if we want to just exclude this.